--- a/apps/websocket/wssServer.cjs
+++ b/apps/websocket/wssServer.cjs
@@ -653144,8 +653144,6 @@
     } : {},
     ...authProviders.includes("ldap") ? {
       AUTH_LDAP_URI: external_exports.string().url(),
-      AUTH_LDAP_BIND_DN: external_exports.string(),
-      AUTH_LDAP_BIND_PASSWORD: external_exports.string(),
       AUTH_LDAP_BASE: external_exports.string(),
       AUTH_LDAP_SEARCH_SCOPE: external_exports.enum(["base", "one", "sub"]).default("base"),
       AUTH_LDAP_USERNAME_ATTRIBUTE: external_exports.string().default("uid"),
@@ -792041,7 +792039,7 @@
 init_common();
 var logger67 = createLogger({ module: "websocketMain" });
 var wss = new import_websocket_server.default({
-  port: 3001
+  port: Number(process.env.PORT_WEBSOCKET) || 3003,
 });
 var handler2 = applyWSSHandler({
   wss,
--- a/apps/tasks/tasks.cjs
+++ b/apps/tasks/tasks.cjs
@@ -724389,8 +724389,6 @@
     } : {},
     ...authProviders.includes("ldap") ? {
       AUTH_LDAP_URI: external_exports.string().url(),
-      AUTH_LDAP_BIND_DN: external_exports.string(),
-      AUTH_LDAP_BIND_PASSWORD: external_exports.string(),
       AUTH_LDAP_BASE: external_exports.string(),
       AUTH_LDAP_SEARCH_SCOPE: external_exports.enum(["base", "one", "sub"]).default("base"),
       AUTH_LDAP_USERNAME_ATTRIBUTE: external_exports.string().default("uid"),
